### PR TITLE
Ability to select state

### PR DESCRIPTION
**Pull request by saipramod**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -61,12 +86,18 @@ export class App extends React.Component<{}, AppState> {
...
                         contact={this.state.contact}
+                        fipsKeyForState={this.state.fipsKeyForState}
                     />
                 </div>
             </div>
         );
     }
 
+    private setFipsKeyForState(fipsKeyForState: string) {
+        console.log(fipsKeyForState);
```
Comment (lzl124631x) [inline src/App.tsx:97]: remove console.log

File: src/stackedchart/StackedChart.tsx [hunk 2]
```diff
@@ -101,10 +127,13 @@ export default class StackedChart extends React.Component<
         const data = await getStackedChart({
             contact: this.props.contact,
             type: this.props.type,
+            fipsKeyForState: this.props.fipsKeyForState,
         });
-        if (data != null) {
+        if (data != null && data[0] != null && data[0].xAxisData.length > 0) {
```
Comment (lzl124631x) [inline src/stackedchart/StackedChart.tsx:132]: if js/ts, let's use `===` instead of `==`

File: src/stackedchart/StackedChart.tsx [hunk 3]
```diff
@@ -101,10 +127,13 @@ export default class StackedChart extends React.Component<
...
             const options = optionsDeletgate(toHighChartData(data[0]));
             this.setState({ options });
+        } else {
+            this.setState({ displayNoDataMessage: true });
```
Comment (lzl124631x) [inline src/stackedchart/StackedChart.tsx:136]: Is this possible? I think all the states must have data

**Comment by lzl124631x** [comment]
Please resolve the conflict.

File: src/stackedchart/StackedChart.tsx (hunk 3 above)
Comment (saipramod) [reply src/stackedchart/StackedChart.tsx:136]: yes i checked.

File: src/stackedchart/StackedChart.tsx (hunk 2 above)
Comment (saipramod) [reply src/stackedchart/StackedChart.tsx:132]: != null, will check both `undefined` and `null`. !== null would check only for null.

File: src/stackedchart/StackedChart.tsx (hunk 3 above)
Comment (saipramod) [reply src/stackedchart/StackedChart.tsx:136]: should i remove this logic or leave it handle if it occurs in the future.

File: src/App.tsx [hunk 4]
```diff
@@ -1,18 +1,95 @@
...
+    { text: "South Dakota ", key: "SD" },
+    { text: "Tennessee ", key: "TN" },
+    { text: "Texas ", key: "TX" },
+    { text: "Utah ", key: "UT" },
+    { text: "Vermont ", key: "VT" },
+    { text: "Virginia ", key: "VA" },
+    { text: "Washington ", key: "WA" },
+    { text: "West Virginia ", key: "WV" },
+    { text: "Wisconsin ", key: "WI" },
+    { text: "Wyoming ", key: "WY" },
```
Comment (lzl124631x) [inline src/App.tsx:57]: nit: let's trim the trailing spaces?

File: src/App.tsx [hunk 5]
```diff
@@ -1,18 +1,95 @@
...
+];
+const typeOptions = [
+    { key: "hosp_need", text: "Bed" },
```
Comment (lzl124631x) [inline src/App.tsx:60]: these are in constants.ts

**Pull request merged**